docs/products/mysql/howto/max-number-of-connections.rst

```diff
@@ -0,0 +1,52 @@
+Calculate the maximum number of connections for MySQL
+=====================================================
+
+The number of simultaneous connections in Aiven for MySQL depends on the usable memory on the server. The usable memory is the total memory on the node minus the operating system and management overhead. This overhead is currently estimated as 350 MiB (≈ 0.34 GiB). 
+
+It can be estimated as:
+
+.. math::
+
+    {usable\_memory} = {total\_memory\_on\_the\_node - management\_overhead} 
+
+To calculate the maximum number of connections, we need to check the value of the usable memory. For this estimation, the usable memory value is **rounded down** to the nearest integer value of GiB. 
+
+**For plans under 4 GiB of usable memory**, you can estimate as
+
+.. note::
+    Independent of the plan, an ``extra_connection`` with a value of ``1`` will be added for the system process.
```

Comment:
Can we move this note above the `**For plans` line? That would make it feel more general.

Otherwise, this seems perfect to me.

Reply:
yes, we can. I also think it makes more sense tbh. Thank you!